docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -100,9 +100,30 @@ Refer to the [Dagster Cloud Branch Deployments example](#example-2-dagster-cloud
 
 ### From Dagster configuration
 
-[Configurable Dagster objects](/concepts/configuration/config-schema) - such as ops, assets, resources, I/O managers, and so on - can accept configuration from environment variables.
+[Configurable Dagster objects](/concepts/configuration/config-schema) - such as ops, assets, resources, I/O managers, and so on - can accept configuration from environment variables. Dagster provides a native way to specify environment variables in your configuration. These environment variables are retrieved at launch time, rather than on initialization as with `os.getenv`.
 
-To access an environment variable as part of a Dagster configuration, use the following syntax:
+This approach has a few unique benefits:
+
+- It provides greater observability. You can see which configuration values are sourced from environment variables in the UI.
+- It makes testing easier, since you can provide string values directly to configuration rather than environment variables.
```

Comment:
Another benefit is secrets hiding in the launchpad and anywhere where we display config.